src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -139,9 +139,15 @@
                                   outgoing] :as message}
 
                           tree]
-  (conj (reduce (fn [acc e] (render-block message acc e)) [react/view {}] tree)
-        [react/text {:style (style/message-timestamp-placeholder outgoing)}
-         (str "  " timestamp-str)]))
+  (let [elements (reduce (fn [acc e] (render-block message acc e)) [react/view {}] tree)
+        timestamp [react/text {:style (style/message-timestamp-placeholder outgoing)}
+                   (str "  " timestamp-str)]
+        last-element (peek elements)]
+    (if (= react/text-class (nth last-element 0))
```

Comment:
Can we use `first` instead of `nth` here?

Reply:
I benchmarked `first` vs `nth` and roughly `nth` is 30% faster, and being a critical path I would be inclined to keep `nth` (although probably at some point we might go for `aget` as it's much much faster, roughly 90% faster then first), but don't mind either ways

Reply:
Oh wow, thanks for important information! Can you then please include a short comment with the performance rationale?

Reply:
sure!